AspectJ-gradle/src/main/kotlin/com/archinamon/api/AspectJCompileTask.kt

```diff
@@ -67,9 +67,6 @@ internal open class AspectJCompileTask: AbstractCompile() {
             task.aspectJWeaver = AspectJWeaver(project)
 
             task.source(sources)
-            task.classpath = classpath()
-
-            findCompiledAspectsInClasspath(task, config.includeAspectsFromJar)
```

Comment:
We could have no javac step (when using pure kotlin/groovy without java sources), so in this case we will have empty classpath. Not good.